pkg/webhooks/sdnmigration/sdnmigration.go

```diff
@@ -45,6 +46,12 @@ type NetworkConfigWebhook struct {
 
 // Authorized will determine if the request is allowed
 func (w *NetworkConfigWebhook) Authorized(request admissionctl.Request) admissionctl.Response {
+	// We are doing this check to ensure that hive can trigger the
+	// migration process. Please refer to OSD-21711 for full explanation.
```

Comment:
Can you please explain some more here? This OSD card is private so not everyone can see the detai.s.

Reply:
Done